storage/badger/chunkDataPacks.go

```diff
@@ -36,6 +37,13 @@ func (ch *ChunkDataPacks) Remove(chunkID flow.Identifier) error {
 	return nil
 }
 
+func (ch *ChunkDataPacks) BatchStore(c *flow.ChunkDataPack, batch storage.BatchStorage) error {
+	if writeBatch, ok := batch.(*badger.WriteBatch); ok {
+		return operation.BatchInsertChunkDataPack(c)(writeBatch)
```

Comment:
The `Store` also handles conflict (with `RetryOnConflict`) in case there is race condition in writing, as well as skipping duplications. 

Would BatchWrite run into transaction confliction? I think it shouldn't since we are purely upserting the data without reading anything. Just to double check

Reply:
No. Documentation on `WriteBatch` class states `Due to the nature of SSI guaratees provided by Badger,
// blind writes can never encounter transaction conflicts (ErrConflict).`